### DISCUSSION TITLE
使用Cloudreve [AWS S3 存储策略] 挂载 Alist S3（对象存储）失败

### DISCUSSION THREAD

**Discussion by ysn2382617094wsw** [AlistGo/alist#6886] [Q&A]
上传文件后显示：
无法解析响应: Unexpected token '<', "<html> <h"... is not valid JSON (<html> <head><title>504 Gateway Time-out</title></head> <body> <center><h1>504 Gateway Time-out</h1></center> <hr><center>nginx</center> </body> </html>       )

测试过Alist S3功能 使用 s3browser 测试过Alist S3是正常的!

**Comment by lonely-4** [comment]
我是这样的：
![Screenshot_20241115_230501_com lemurbrowser exts](https://github.com/user-attachments/assets/74bd4ed0-e152-425e-801e-b6e84f308f66)
